src/components/Switcher/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import Image from 'next/image';
+import Style from './ThemeSwitcher.module.scss';
+import { useTheme } from '@/contexts/ThemeContext/ThemeContext';
+
+export default function ThemeSwitcher() {
+  const { theme, toggleTheme } = useTheme();
+  return (
+    <div className={Style.switcher}>
+      <button className={Style.switcher__button} onClick={() => toggleTheme('light')}>
+        <Image
+          className={Style.switcher__icon}
+          src={'/sun.svg'}
+          alt='sun'
+          width={30}
+          height={30}
+          style={{
+            backgroundColor: theme === 'light' ? 'var(--primary-color)' : 'transparent',
```

Comment:
Never set inline styles, you can only set css-variable in inline styles. Instead of using a ternary operator to change color, you must change the class name, like `switcher__icon—selected `